src/ui.js

```diff
@@ -239,6 +272,29 @@ export function buildSettingsModal(tracks, opts, finishCallback) {
             value=${opts.lineOptions.weight}>
     </span>
 
+    <span class="form-row">
```

Comment:
Maybe these should be grouped under "image settings" or so? Whatever you think makes it more clear